package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/node-server-sdk",
-  "version": "3.6.0",
+  "version": "3.6.1",
```

no need; sufficient that its removed. We'll publish again soon but your call.

```suggestion
  "version": "3.6.0",
```

---

Let's still bump, but we don't need to cut a release right easy. Should still bump the patch# here since there's a fix/patch?